core/database_arango.py

```diff
@@ -588,7 +584,7 @@ def neighbors(
 
         aql = f"""
         FOR v, e, p IN @min_hops..@max_hops {direction} @extended_id @@graph
-
+          OPTIONS {{ uniqueVertices: "path" }}
```

Comment:
for my own understanding, no impact on using this option?

Reply:
Yes, but the impact is intended.

This will just make sure that when traversing the graph, a same vertice isn't traversed more than once per path (in a connected graph, this can be useful to avoid infinite paths :D)

Check out the OPTIONS section here: https://docs.arangodb.com/3.12/aql/graphs/traversals/#working-with-named-graphs